docker/nginx/nginx.conf

```diff
@@ -1,88 +1,82 @@
-
-    root /var/www/html;
-
-    location ~ /.well-known {
-      allow all;
-    }
-
-location /css {
-    alias  /openhabcloud/openhab-cloud/public/css;
-    }
-location /js {
-    alias /openhabcloud/openhab-cloud/public/js;
-    }
-location /img {
-    alias /openhabcloud/openhab-cloud/public/img;
-    }
-location /bootstrap {
-    alias /openhabcloud/openhab-cloud/public/bootstrap;
-    }
-location /font-icons {
-    alias /openhabcloud/openhab-cloud/public/font-icons;
-    }
-location /fonts {
-    alias /openhabcloud/openhab-cloud/public/fonts;
-    }
-location /js-plugin {
-    alias /openhabcloud/openhab-cloud/public/js-plugin;
-    }
-location /downloads {
-    alias /openhabcloud/openhab-cloud/public/downloads;
-    }
-
-location ~ ^/(socket.io|rest|images|static|rrdchart.png|chart|openhab.app|WebApp|CMD|cometVisu|proxy|greent|jquery|classicui|ui|basicui|doc|start|icon|habmin|remote|habpanel|ifttt/v1/actions/command){
-    proxy_pass http://app_server;
-    proxy_redirect off;
-    proxy_http_version 1.1;
-    proxy_set_header Host $host;
-    proxy_set_header Upgrade $http_upgrade;
-    proxy_set_header Connection "upgrade";
-    proxy_set_header X-Real-IP $remote_addr ;
-    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for ;
-    #proxy_set_header X-Forwarded-Proto https;
-}
-
-location / {
-    proxy_pass http://app_server;
-    proxy_redirect off;
-    proxy_http_version 1.1;
-    proxy_set_header Host $host;
-    proxy_set_header X-Real-IP $remote_addr ;
-    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for ;
-    #proxy_set_header X-Forwarded-Proto https;
-}
-
-#error_page 404 /404.html;
-
-# redirect server error pages to the static page /50x.html
-#
-#error_page 500 502 503 504 /50x.html;
-#location = /50x.html {
-#root html;
-#}
-}
+	server {
+	    listen 80 default_server;
+	    listen [::]:80 default_server;
+	
+	    charset utf-8;
+	
+	    access_log /var/log/nginx/openhab.org-access.log;
+	    error_log /var/log/nginx/openhab.org-error.log;
+	    client_max_body_size 300m;
+	
+	    root /opt/openhabcloud;
+	
+	    location ~ /.well-known {
+	      allow all;
+	    }
+	
+		location /css {
+	    	alias  /opt/openhabcloud/public/css;
+	    }
+		location /js {
+	    	alias /opt/openhabcloud/public/js;
+	    }
+		location /img {
+	    	alias /opt/openhabcloud/public/img;
+	    }
+		location /bootstrap {
+	    	alias /opt/openhabcloud/public/bootstrap;
+	    }
+		location /font-icons {
+	    	alias /opt/openhabcloud/public/font-icons;
+	    }
+		location /fonts {
+	    	alias /opt/openhabcloud/public/fonts;
+	    }
+		location /js-plugin {
+	    	alias /opt/openhabcloud/public/js-plugin;
+	    }
+		location /downloads {
+	    	alias /opt/openhabcloud/public/downloads;
+	    }
+	
+		location ~ ^/(socket.io|rest|images|static|rrdchart.png|chart|openhab.app|WebApp|CMD|cometVisu|proxy|greent|jquery|classicui|ui|basicui|doc|start|icon|habmin|remote|habpanel|ifttt/v1/actions/command){
```

Comment:
cool..this must be my text wrangler mix with eclipse editor :-)
Will correct that..